packs/equipment/cloak-of-elvenkind-greater.json

```diff
@@ -34,7 +34,23 @@
             "title": "Pathfinder Core Rulebook"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "cloak-of-elvenkind-greater",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "cloak-of-elvenkind-greater"
+                ],
+                "selector": "stealth",
+                "type": "item",
+                "value": 1
+            }
+        ],
```

Comment:
At the least it should be a +2, but also I don't know that the toggle is needed. I mean practically you'll just always have it on.